Ziti.NET.Standard/src/OpenZiti/ZitiMFAService.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace OpenZiti
+{
+	public enum MFAOperationType
+	{
+		MFA_AUTH_STATUS,
+		ENROLLMENT_VERIFICATION,
+		ENROLLMENT_REMOVE,
+		ENROLLMENT_CHALLENGE
+	}
+
+	public struct ZitiMFAEnrollment
+	{
+		public bool isVerified;
+		public string[] recoveryCodes;
+		public string provisioningUrl;
+	}
+
+	public class ZitiMFAService
```

Comment:
refactor and i'll review after

Reply:
I think I will move the MFA operations to the ZitiIdentity class, so the users can call the MFA operations using the ZitiIdentity object. Is that okay?